bzt/modules/aggregator.py

```diff
@@ -398,7 +396,7 @@ def datapoints(self, final_pass=False):
         for datapoint in self._calculate_datapoints(final_pass):
             current = datapoint[DataPoint.CURRENT]
             self.__merge_to_cumulative(current)
-            datapoint[DataPoint.CUMULATIVE] = copy.deepcopy(self.cumulative)
```

Comment:
Deepcopy is for reason. Some modules modify mutable datapoint[DataPoint.CUMULATIVE]  and it screws stats.